app/controllers/ops_controller.rb

```diff
@@ -552,6 +555,15 @@ def settings_replace_right_cell(nodetype, presenter, r)
       else
         @right_cell_text = _("Editing %{model} \"%{name}\"") % {:name => @category.description, :model => "Category"}
       end
+    when "ltme" # label tag mapping edit
+      # when editing/adding label tag mapping in settings tree
+      presenter.update(:settings_label_tag_mapping, r[:partial => "label_tag_mapping_form"])
+      if !@lt_map
+        @right_cell_text = _("Adding a new Mapping")
+      else
+        @right_cell_text = _("Editing %{model} \"%{name}\"") % {:name  => @lt_map.label_name,
```

Comment:
The `_("... %{model} ...")` convention appears a lot in UI code but is mostly deprecated.
If you do use it, the model name should be translated using `:model => ui_lookup(...)`, but that's only justified when  same code handles many models; here it's constant so just write it inside the translation string e.g.:

    @right_cell_text = _("Editing tag mapping from label \"%{name}\"") % {:name  => @lt_map.label_name}

[see https://github.com/ManageIQ/guides/blob/master/i18n.md if you want to learn more about translation]